lib/routes/index.js

```diff
@@ -6,8 +6,10 @@ var routes = {
 
 
 exports.routes = function (version) {
-  if (routes['v' + version]) {
-    return routes['v' + version];
+  var matches = /(\d+)\.\d+/.exec(version);
```

Comment:
I would prefer something like major = verson`/(\d+)(\.\d+)?/` since until now we used to just pass `2` in the version.

Reply:
I don't get what you mean here. 
Do you mean that you'd like to have `major` possibly equal to something like `2.7` and have the routes match that? Which would lead to having multiple routes definitions like so:
```
var routes = {
  'v2': require('./v2'),
  'v2.1': require('./v2.1'),
  'v2.2': require('./v2.2'),
};
```

Or do you mean something else?

Reply:
I think I get it. You want to use  `/(\d+)(\.\d+)?/` instead so that if we use `2` it matches correctly against the regex, which is not the case with the current one. I'll go with that.

Reply:
Yes that was it :+1:

Reply:
2 is actually the default value that the lib uses when a custom version isn't defined, so this would break the lib for all existing users until they start using the version option.